doc/source/project/issues.rst

```diff
@@ -217,6 +217,20 @@ special purpose.
   bug affects the actual current Galaxy development branch and isn't a usage
   issue, a previously fixed issue, etc..
 
+- ``highlight`` is a tag that indicates at least one contributor thinks this
+   pull request should be highlighted in some way in the relevant release's
+   release notes. The person assembling release notes has final say about
```

Comment:
I wonder, do we need to include this last sentence? I think it's clear that the final contents of the release notes is determined by the release team (the release manager, or a team, a committee, etc. - whatever it is), and not by the person who added the label. To me, the text seems to be a little too specific and prescriptive: we'd have to edit this page every time we change our approach. Same goes for the user label below.

Reply:
Which is OK, better that we update this if needed then just leaving it vague ?
I'll merge this but feel free to open a follow up